runtime/fake-pec-factory.js

```diff
@@ -0,0 +1,20 @@
+// @license
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+"use strict";
+
+const InnerPec = require('./inner-PEC.js');
+const OuterPec = require('./outer-PEC.js');
+const MessageChannel = require('./message-channel.js');
+const Loader = require('./loader.js');
+
+module.exports = function(loader, id) {
+  var channel = new MessageChannel();
+  new InnerPec(channel.port1, `${id}:inner`, new Loader());
+  return new OuterPec(channel.port2, `${id}:outer`);
+};
```

Comment:
we want to maintain the ability to run in-process pecs in the browser. Add a TODO to re-expose this or add a new one for the browser?